test.css

```
@@ -0,0 +1,87 @@
+body{
+  background-color:pink;
+
+}
+div{
+    display:flex;
+    justify-content:center;
+}
+h3{
+    display:flex;
+    justify-content:center;
+    margin-left:50px;
+    margin-right:50px;
+    margin-top:30px;
+    font-size:20px;
+}
+p{
+    display:flex;
+    justify-content:center;
+}
+.score{
+    border-style:solid;
+    border-width:2px;
+    padding-left:30px;
+    padding-right:30px;
+    padding-bottom:10px;
+    padding-top:10px;
+    font-size:30px;
+}
+.result{
+  font-size:20px;
+}
+.block{
+  justify-content:center;
+}
+.yourChoice{
+  font-size:100px;
+  margin-top:0px;
+  margin-right:10px;
+}
+.computerChoice{
+  font-size:100px;
+  margin-top:0px;
+  margin-left:10px;
+}

```

✨P2: css에서 내용이 같은 class가 있다면, 한번에 참조할 수 있습니다! 이렇게 하면 코드 작성도 더 빠르고, 수정에도 용이합니다.
```suggestion
.yourChoice, .computerChoice {
  font-size:100px;
  margin-top:0px;
  margin-right:10px;
}
```